test/e2e/harbor_test.go

```diff
@@ -20,6 +23,19 @@ func TestCPackagesHarborInstallSimpleFlow(t *testing.T) {
 	runHarborInstallSimpleFlow(test) // other args as necessary
 }
 
+func TestCPackagesHarborNutanixKubernetes123SimpleFlow(t *testing.T) {
+	framework.CheckCuratedPackagesCredentials(t)
+	test := framework.NewClusterE2ETest(t,
+		framework.NewNutanix(t),
```

Comment:
Need to pass the ubuntu image 
```suggestion
		framework.NewNutanix(t, framework.WithUbuntu123Nutanix()),
```